.gitignore

```diff
@@ -1,2 +1,4 @@
 .DS_Store
 /Manifest.toml
+binary/
```

Comment:
We need to put the binary in `build` folder, just simply rename the `binary` -> `build`, so we could use BB (BinaryProvider) for MAGMA later.

Reply:
I changed the name just now.